src/table/table.scss

```diff
@@ -117,12 +119,15 @@
 }
 
 @mixin iui-table-body {
-  overflow-x: hidden;
+  overflow-x: auto;
+  overflow-x: overlay;
   overflow-y: scroll;
   overflow-y: overlay;
   display: flex;
   flex-direction: column;
   flex-grow: 1;
+  // Allows columns with static width to take more space than the table width
+  align-items: flex-start;
```

Comment:
If you change this to `center`, you won't need to specify it again for empty table